python/src/GaussianNonLinearCalibration_doc.i.in

```diff
@@ -33,18 +33,31 @@ dimension of the model and the number of observations.
 The prior distribution of the parameter is a :class:`~openturns.Normal`.  
 
 The posterior distribution of the parameter is :class:`~openturns.Normal`. 
-By default, the posterior distribution is evaluated based on a linear approximation 
-of the model at the optimum. 
-This corresponds to using the :class:`~openturns.GaussianLinearCalibration` at the optimum,
-and is named *Laplace approximation* in the bayesian context.
-However, if the key `GaussianNonLinearCalibration-BootstrapSize` in the :class:`~openturns.ResourceMap`
-is set to a nonzero positive integer, then a bootstrap resampling of the observations
-is performed and the posterior distribution is based on a :class:`~openturns.KernelSmoothing`
-of the sample of boostrap optimum parameters.
+Depending on the value of the `GaussianNonLinearCalibration-BootstrapSize` key 
+in the :class:`~openturns.ResourceMap`, the parameters are estimated differently.
+
+- If the key `GaussianNonLinearCalibration-BootstrapSize` in the :class:`~openturns.ResourceMap`
```

Comment:
```suggestion
- If `GaussianNonLinearCalibration-BootstrapSize`
```
There is already a mention of the fact that this is a ResourceMap key right above, so I suggest this to make the text easier to read.